src/main.c

```diff
@@ -55,27 +55,31 @@ int main (int argc, char* argv[])
 	logg("########## FTL started on %s! ##########", hostname());
 	log_FTL_version(false);
 
+	// Process pihole-FTL.conf
```

Comment:
Why did you move reading the conf file before initializing the shared memory?

Reply:
This has been mentioned in the commit message of https://github.com/pi-hole/FTL/pull/1464/commits/fc303aeba52cd2e742ea6cc2b62e3d6c8a5f13ac. Sorry, I should have included this in the PR description:

> Read FTL config *before* initializing shared memory to ensure we are aware of a possible DEBUG_SHMEM flag when creating the shared memory segments

Without this, any `DEBUG_SHMEM` code was useless in `init_shmem()`.